tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2040,23 +2040,23 @@ declare namespace ts {
         getNonNullableType(type: Type): Type;
         getTypeArguments(type: TypeReference): readonly Type[];
         /** Note that the resulting nodes cannot be checked. */
-        typeToTypeNode(type: Type, enclosingDeclaration?: Node, flags?: NodeBuilderFlags): TypeNode | undefined;
+        typeToTypeNode(type: Type, enclosingDeclaration: Node | undefined, flags: NodeBuilderFlags | undefined): TypeNode | undefined;
```

Comment:
Definitely shouldnt be breaking api for 3.9

Reply:
> @typescript-bot cherry-pick this to release-3.9 (once the PR is up, I'll edit it to undo the API changes)

I know